sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -144,7 +184,7 @@ func NewManager(
 		done:              concurrency.NewSignal(),
 		connectionsByHost: make(map[string]*hostConnections),
 		clusterEntities:   clusterEntities,
-		flowUpdates:       make(chan *central.MsgFromSensor),
```

Comment:
Why is this being removed? There should be no changes to the network flow updates.

Reply:
It's not moved, I'm using the same channel to send the PLoP messages alongside the NetwrokFlow ones, so I simply renamed the channel in order to make it clear that this is no longer a flow exclusive channel, does that make sense?

Reply:
Okay